### DISCUSSION TITLE
Is there a way to have Web Search Navigator work with other Search Engines (not Listed in the Options)?

### DISCUSSION THREAD

**Discussion by Cr8zyIvan** [infokiller/web-search-navigator#558] [Q&A]
Is there a way to "Add Search Engines" to a "Custom Search Engines List" for Web Search Navigator?
I'm using Firefox.

I'm currently exploring alternative Search Engines. For right now, I would like to make Web Search Navigator function on the DuckDuckGo and Ecosia Search Engines. I would also like to use Web Search Navigator on any other Search Engine Website I'm going to test. Just to be clear, I mean Search Engines that are not already listed in the "EXPERIMENTAL: Alternative search engines > Help" section of Web Search Navigator's Options (Brave Search, Startpage, YouTube, etc.)

All Websites are Coded differently, and perhaps this is not a feasible ask. But if you don't ask, the answer is always no. Perhaps also, there is a partial solution, where Web Search Navigator can work only on certain Search Engines / Websites.

Any insight would be greatly appreciated. Cheers!

**Comment by infokiller** [comment]
Hey, the short answer is no. In general, every search engine requires new code to support it (as documented in https://github.com/infokiller/web-search-navigator#adding-a-new-search-engine), since it has a different HTML structure.

**Reply by Cr8zyIvan** [reply]
Cool. Thank you for the reply!
